js/src/main/scala/crystal/react/hooks/UseStateView.scala

```diff
@@ -0,0 +1,79 @@
+package crystal.react.hooks
+
+import cats.syntax.all._
+import crystal.react.View
+import japgolly.scalajs.react._
+import japgolly.scalajs.react.hooks.CustomHook
+import japgolly.scalajs.react.util.DefaultEffects.{ Sync => DefaultS }
+
+import scala.collection.immutable.Queue
+import scala.reflect.ClassTag
+
+object UseStateView {
+  def hook[A: Reusability: ClassTag]: CustomHook[A, View[A]] =
```

Comment:
Why do you need a `ClassTag`?

Reply:
It's required to build the `Reusable` instance in `useStateWithReuse`.

That said, I think we may not need the `WithReuse` part here (there's universal `Reusability[View[A]]` as long as there's `Reusability[A]`. I'll attempt to convert this to a regular `useState`.